core/src/openid.cpp

```diff
@@ -246,6 +246,12 @@ namespace irods::http::openid
 			key = jwt::base::decode<jwt::alphabet::base64url>(
 				jwt::base::pad<jwt::alphabet::base64url>(realm_secret->get_ref<const std::string&>()));
 		}
+		else if (auto secret{irods::http::globals::oidc_configuration().find("non_standard_id_token_secret")};
```

Comment:
nonstandard?   without the space/underscore?   hmmm.

Reply:
Done.